packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/duration-fields.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import { InputField, SelectField } from '.';
+import { TDropdownItems, TSelectsFieldNames } from '../quick-strategy.types';
+import { TDurationFields } from './components.types';
+
+const DurationFields = ({
+    id,
+    is_mobile,
+    field_name,
+    idx,
+    dropdown_list,
+    selected_value,
+    label,
+    select_value,
+    className,
+    is_able_disabled,
+    values,
+    selected_trade_type,
+    selected_symbol,
+    errors,
+    setFieldValue,
+    onChangeDropdownItem,
+    onHideDropdownList,
+    onScrollStopDropdownList,
+    handleChange,
+    onChangeInputValue,
+    setCurrentFocus,
+}: TDurationFields) =>
+    id === 'duration-unit' ? (
```

Comment:
here too

Reply:
@rupato-deriv answer will be the same as above